Sources/TuistCache/Cache/FileClient.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistSupport
+
+enum FileClientError: LocalizedError, FatalError {
```

Comment:
This file is a good candidate to be part of `TuistSupport`. It's Tuist-agnostic and we might need to reuse it from other places.

Reply:
Moved